### ISSUE TITLE
Analysis of space in the robot Day #11

### ISSUE THREAD

**Issue by WROMagnet** [WROMagnet/Magnet-Robots#88]
**Today, September 10, we analyzed and were thinking of putting the ultrasonic sensors connected to an Arduino UNO and the Arduino connected to the PRIZM but we did not think about the space.**

**Comment by WROMagnet** [comment]
- We were thinking of putting the Arduino UNO under the Prizm, but we also thought of putting an adapter on the Arduino UNO so that it has connectors for the ultrasonic sensors. So if you put the adapter on, you can't put the Prizm under it.

- We have to think about how to make more space or how to eliminate things that are not necessary.

**Comment by WROMagnet** [comment]
![Image](https://github.com/user-attachments/assets/c166d6f7-4362-486c-a32d-2a2d9baf0214)

- Since this is the model of the Robot @LuisriosWRO3 thought about placing the prizm vertically to make space in the center for the Arduino UNO. But we think that the Prizm vertically in a corner will unbalance the Robot and there will be one side closer to the floor than the other.

-I (@DanielRomanRodriguez ) gave an idea to replace the prizm with an Arduino UNO but with an adapter called MOTOR SHIELD. It is an Arduino UNO adapter that makes it possible for an Arduino to control a DC motor greater than 5 Volts.

![Image](https://github.com/user-attachments/assets/83d61b1d-94c9-4b42-9355-c681121efb12)

- This adapter runs on a voltage individual to that of the Arduino. In other words, a voltage of 11V or 12V is needed to make the DC motors run fast but it uses the Arduino's mind.

- This adapter also has connectors for continuous servo motors such as 180 degrees. By putting an Arduino with this adapter and putting 11.5V into the adapter, it would do exactly the same function as the Prizm, only it takes up less space and the codes are without any type of library.

**Comment by WROMagnet** [comment]
- This adapter works in a bit of a weird way, but it's not complicated at all, you just have to learn which pins are used to accelerate the pressing and the speed and after that it's easy.

- We got this adapter in one of the workshop boxes and I was curious and looked for information and it turned out that we were going to use it in the end.

- When I gave the idea to the master, he didn't pay much attention to it because we were already used to the Prizm, but he told me that if I made it run just like the Prizm it would accept the change. So, I quickly started working on it.
